pages/community/team.mdx

```diff
@@ -38,12 +37,24 @@
 - Protocol development
 - Contributions to specifications
 
-[Konrad Staniec](https://github.com/KonradStaniec)
+[Daniel Sobol](https://github.com/mynameisdaniil)
 - Full-time developer on Fluffy client
-- Minor contributions to specifications
 
 ## 3rd Party
```

Comment:
Maybe "3rd party and prior contributors"? since jacob isn't really 3rd party but we still want to give him credit for his prior contributions?

Reply:
I added a ``Prior Contributors`` section is that what you meant?